packages/openai/src/responses/openai-responses-language-model.test.ts

```diff
@@ -89,6 +89,32 @@ describe('OpenAIResponsesLanguageModel', () => {
                     type: 'output_text',
                     text: 'answer text',
                     annotations: [],
+                    logprobs: [
+                      {
+                        token: 'Hello',
+                        logprob: -0.0009994634,
+                        top_logprobs: [
+                          {
+                            token: 'Hello',
+                            logprob: -0.0009994634,
+                          },
+                          {
+                            token: 'Hi',
+                            logprob: -0.2,
+                          },
+                        ],
+                      },
+                      {
+                        token: '!',
+                        logprob: -0.13410144,
+                        top_logprobs: [
+                          {
+                            token: '!',
+                            logprob: -0.13410144,
+                          },
+                        ],
+                      },
+                    ],
```

Comment:
recommend making configurable and only adding in the logprobs test to ensure realistic payloads and the non logprobs case coverage

Reply:
pattern is to make it an optional argument and only pass it in in the specific test case

Reply:
oh i was thinking there would be a prepareRequest function or similar (many tests have it) but it does not seem to exist here. might be worth introducing